packages/zosfiles/__tests__/__unit__/methods/list/List.unit.test.ts

```diff
@@ -173,15 +173,18 @@ describe("z/OS Files - List", () => {
             const memberNames = ["m1", "m2"];
             const shuffledAsciiChars = String.fromCharCode(...Array.from(Array(256).keys()).sort(() => Math.random() - 0.5));
             for (let i = 0; i < 32; i++) {
-                // Exclude double quotes for now
-                memberNames.push(shuffledAsciiChars.slice(i * 8, (i + 1) * 8).replace('"', ''));
+                memberNames.push(shuffledAsciiChars.slice(i * 8, (i + 1) * 8));
             }
             expectStringSpy.mockResolvedValueOnce(`{"items":[\n` +
                 memberNames.map((memName) => `  {"member":"${memName}"}`).join(",\n") + `\n` +
                 `],"returnedRows":${memberNames.length},"JSONversion":1}`);
 
             const expectedListApiResponse = {
-                items: memberNames.map((memName) => ({ member: memName.replace((List as any).CONTROL_CHAR_REGEX, "\ufffd") })),
+                items: expect.arrayContaining([
+                    { member: "m1" },
+                    { member: "m2" },
+                    { member: expect.stringMatching(/… (\d+) more members/) }
```

Comment:
Can this break --rfj output?

Reply:
Alternatively, should we consider not adding this as an additional member (since it isn't technically a valid member), and trigger logic to warn if the list of members doesn't match the expected number of members that is returned in the `returnedRows` property?